presentation/src/main/java/org/cryptomator/presentation/ui/dialog/HubLicenseUpgradeRequiredDialog.kt

```
@@ -0,0 +1,43 @@
+					dialog?.dismiss()
+					callback?.onHubLicenseUpgradeRequiredDialogFinished()
+					true
+				} else {
+					false
+				}
+			}
+		return builder.create()
+	}
+
+	public override fun setupView() {
+	}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)